sections/sections-app-util/src/bundle/sections.properties

```diff
@@ -1,4 +1,6 @@
 #Tue Jan 25 16:42:46 CET 2011
+options_confirmInternal=Manually manage avoids groups updates for registration system. Do you want to continue?
```

Comment:
Manually manage will prevent group membership updates from an automated enrollment system. Do you want to continue?